### ISSUE TITLE
Make projects on the left sidebar clickable by middle mouse button

### ISSUE THREAD

**Issue by AlexWayfer** [cirruslabs/cirrus-ci-docs#671]
# Expected Behavior

I want to open different projects from the left sidebar on cirrus-ci.com in new tabs by clicking via middle mouse button on them.

# Real Behavior

Nothing.

# Related Info

- [x] Website issue
  - Link to page: https://cirrus-ci.com/

![image](https://user-images.githubusercontent.com/6510020/87444477-84965b00-c5ff-11ea-9466-29e7dc378159.png)

**Comment by AlexWayfer** [comment]
Thank you! :clap:

**Comment by RDIL** [comment]
@fkorotkov here’s why `a` tags would be more effective - they are meant to handle these things on the browser end.  It will be easier to fix by just switching over.

**Comment by fkorotkov** [comment]
Agreed, need to investigate how tags are playing along with the SPA nature of the front end (if there is a refresh when you click on a task from a build page or simply a GraphQL call like right now).

**Comment by RDIL** [comment]
I’ll try migrating this to an `a` tag later today, and if it all works as expected then maybe we can look in to switching the rest of the codebase.

**Comment by RDIL** [comment]
Update on this:

So I tried implementing it, but it messed up the styling, and made React fail the DOM tree tests, because `a` tags can't be the parent of a table row.  

So yeah, this sadly didn't work.
